### DISCUSSION TITLE
Bookmarklet disappearing

### DISCUSSION THREAD

**Discussion by derixithy** [Kovah/LinkAce#729] [General]
When i want to bookmark this page https://aniwave.to/home the bookmarklet page show for a quick second and then disappears. But if i try for example https://google.com the popup stays on screen.

Anyone else has this problem, is this my browser or a bug?

**Comment by Kovah** [comment]
Unfortunately, websites can mess with the way the bookmarklet is working. A good example is the Internet Archive, where the bookmarklet doesn't work at all because the override browser standards. 🤷
